packages/frontend/editor-ui/src/features/settings/sso/styles/sso-form.module.scss

```diff
@@ -38,6 +38,11 @@
 	}
 }
 
+.checkboxGroup label > *:first-child {
```

Comment:
Just another nit: I usually avoid these selectors (bound to structure)

But actually, if we went with what we usually use here, it would give us more work after the design system refactoring.

```scss
.checkboxGroup :deep(.el-checkbox__input) {
  vertical-align: text-top;
}
```

Reply:
Agreed, this kind of selector is likely to break without noticing. I just tried this one locally, but the style isn't applied. Not sure why. It looks correct to me.